server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityStatusAction.java

```diff
@@ -75,13 +74,15 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
+    // fail-fast
+    userSession.checkLoggedIn();
```

Comment:
"Anyone" could be granted admin permission on a project, it's not a good idea to enforce user must be logged in